### ISSUE TITLE
Implement XAI Method: Integrated Gradient

### ISSUE THREAD

**Issue by muellerdo** [frankkramer-lab/aucmedi#25]
https://github.com/experiencor/deep-viz-keras
https://keras.io/examples/vision/integrated_gradients/
https://github.com/hiranumn/IntegratedGradients

https://arxiv.org/abs/1703.01365
